python/pyspark/sql/tests/connect/test_parity_dataframe.py

```diff
@@ -54,6 +54,10 @@ def test_cached_property_is_copied(self):
     def test_toDF_with_schema_string(self):
         super().test_toDF_with_schema_string()
 
+    @unittest.skip("Spark Connect does not support mergeInto at the moment.")
```

Comment:
Are we planning to support Spark Connect?

Reply:
> The support for this API in Spark Connect Python API will be added later by https://github.com/apache/spark/pull/46960.

I guess so from the PR description.